libcaf_net/test/detail/rfc6455.cpp

```diff
@@ -60,6 +60,40 @@ CAF_TEST(masking) {
   CAF_CHECK_EQUAL(masked_data, data);
 }
 
+CAF_TEST(no mask key and no data) {
+  std::vector<uint8_t> data{};
```

Comment:
Style: calling the initializer is unnecessary. Looking at the file, I think it's just copy/paste, still wanting to point it out. In general:

```suggestion
  std::vector<uint8_t> data;
```

or

```suggestion
  auto data = std::vector<uint8_t>{};
```